engineapi/backup_monitor.go

```diff
@@ -71,6 +73,9 @@ func NewBackupMonitor(logger logrus.FieldLogger,
 
 	// Call engine API snapshot backup
 	if backup.Status.State == longhorn.BackupStateNew {
+		// put volume recurring jobs/groups information into backup labels and it would be stored in the file `volume.cfg`
+		backup.Spec.Labels[types.VolumeRecurringJobInfoLabel] = volumeRecurringJobInfo
```

Comment:
Check the nullity before adding a new lable.

Reply:
Modified and checked. Thanks